pandas/tests/io/test_gbq.py

```diff
@@ -149,34 +147,33 @@ def mock_read_gbq(sql, **kwargs):
 
 @pytest.mark.single
 class TestToGBQIntegrationWithServiceAccountKeyPath:
-    @classmethod
-    def setup_class(cls):
-        # - GLOBAL CLASS FIXTURES -
-        # put here any instruction you want to execute only *ONCE* *BEFORE*
-        # executing *ALL* tests described below.
-
+    @pytest.fixture()
+    def gbq_dataset(self):
+        # Setup Dataset
         _skip_if_no_project_id()
         _skip_if_no_private_key_path()
 
-        cls.client = _get_client()
-        cls.dataset = cls.client.dataset(DATASET_ID + "1")
+        dataset_id = "pydata_pandas_bq_testing_py31"
```

Comment:
hmm, this is different than the previous, is that right?

Reply:
Previous is here:

https://github.com/pandas-dev/pandas/pull/30478/files#diff-19467f8af166870b0cd441ab9be7ef7cL21-L24

I kept the dataset_id the same as before